rl.go

```diff
@@ -89,7 +89,7 @@ func (rl *rateLimiter) Handler(next http.Handler) http.Handler {
 				windowLen: windowLen,
 				limiter:   limiter,
 			}
-			lhs = append(lhs, lh)
+			lastLH = lh
```

Comment:
You're absolutely right!!

Originally, I wanted to keep the **first** LimitHandler in this `lastLH`.

ref: https://github.com/k1LoW/rl/blob/4bac38375852c77194b1806a64f112dca8de2bd1/rl.go#L159

I will change [the comment](https://github.com/k1LoW/rl/blob/4bac38375852c77194b1806a64f112dca8de2bd1/rl.go#L159) of the New function to go along with this implementation.